.dassie/config/tinymce.yml

```diff
@@ -0,0 +1,13 @@
+default: &default
+  setup: tinymce_nav_safety # onChange event init
+content_block:
+  <<: *default
+  menubar: false
+  toolbar1: styleselect | bold italic | undo redo
+  toolbar2: table | fullscreen | image
+  plugins:
+    - table
+    - fullscreen
+    - image
+custom:
+  <<: *default
```

Comment:
This is for dassie which is not in 2.x, so this file can be removed from this PR.

Reply:
dassie was backported to 2.x sometime in the last few months!